src/main/java/edu/unc/lib/boxc/migration/cdm/services/BoxctronFileService.java

```diff
@@ -47,8 +49,13 @@ public void generateMapping(BoxctronFileMappingOptions options) throws IOExcepti
         assertProjectStateValid();
         ensureMappingState(options);
 
-        // Gather listing of all potential source file paths to match against
+        // Gather list of all potential source file paths to match against
         List<String> candidatePaths = gatherCandidatePaths(getVelocicroptorDataPath(project.getProjectPath()));
+        // Gather list of all source file paths to exclude
+        List<String> exclusionPaths = new ArrayList<>();
```

Comment:
Just a minor efficiency note, since this list is exclusively for lookups, it would be more efficient to use a Set/HashSet since the time for `contains` calls is constant, meaning it won't take any more time to search a set of 10 or 1000000 entries, versus a list where the search time depends on the number of entries. If you're curious https://www.baeldung.com/java-collections-complexity